docs/other/office.md

```diff
@@ -15,22 +15,22 @@ MetaDescription: See how to use the features of Visual Studio Code and integrate
 
 Under the hood, an Office Add-in is just a web app that you can host anywhere. Using a `manifest.xml` file, you tell the Office application where your web app is located and how you want it to appear. The Office application takes care of hosting it within Office.
 
-## Step 1: Get set up
+## Step 1: Create a new Office Add-in project
 
 Follow the instructions to [Create an Office Add-in using any editor](https://developer.microsoft.com/office/blogs/creating-office-add-ins-with-any-editor-introducing-yo-office) to install the necessary prerequisites, and use Yo Office to create a new add-in project. The following table lists the project attributes to select in the Yeoman generator.
 
 | Option | Value |
 |:------|:------|
-|New subfolder | (accept the default) |
+|New subfolder | (Accept the default) |
 | Add-in name | Outlook Add-in |
-| Supported Office application | (select Outlook) |
+| Supported Office application | (Select Outlook) |
 | Create new add-in | Yes, I want a new add-in |
 | Add [TypeScript](https://www.typescriptlang.org/) | No |
 | Choose framework | Jquery |
 
-## Use Visual Studio Code to develop your Office Add-in!
+## Use Visual Studio Code to develop your Office Add-in
 
-Visual Studio Code is a great tool to help you develop your custom Office Add-ins regardless if they are for Outlook, Word, Excel, PowerPoint and run in the web clients, Windows clients, iOS clients or on macOS!
+Visual Studio Code is a great tool to help you develop your custom Office Add-ins for Outlook, Word, Excel, PowerPoint and runs in web clients, Windows clients, iOS clients or on macOS!
```

Comment:
I think this rewrite changes the meaning and implies that VS Code runs in web clients, Windows, iOS, and macOS, when the original sentence it talking about the Office Add-ins.

Reply:
Ah got it - I actually originally interpreted it that way but this makes more sense. Perhaps, it's worth splitting this into two separate sentences.

Visual Studio Code is a great tool to help you develop your custom Office Add-ins for Outlook, Word, Excel, PowerPoint. It also enables you to run your Office Add-ins in web clients, Windows clients, iOS clients or on macOS!